custom/ilsgateway/models.py

```diff
@@ -485,6 +485,17 @@ class Meta:
         app_label = 'ilsgateway'
 
 
+class OneOffTaskProgress(models.Model):
```

Comment:
i realize this is a custom model but i think it's still important to put domain on here and make sure that we filter on domain everywhere it's used